app/controllers/patients_controller.rb

```diff
@@ -314,6 +314,16 @@ def update_status
     patient = current_user.get_patient(params.permit(:id)[:id])
     redirect_to(root_url) && return if patient.nil?
 
+    # update LDE for patient and group members only in the exposure workflow with continuous exposure on (separate from updating monitoring status)
+    if params.permit(:apply_to_group_cm_exp_only)[:apply_to_group_cm_exp_only]
+      ([patient] + (current_user.get_patient(patient.responder_id)&.dependents&.where(continuous_exposure: true, isolation: false) || [])).uniq.each do |member|
+        next unless params[:apply_to_group_cm_exp_only_date].present?
```

Comment:
It might be because the diff view is really crowded with comments right now, but why is this in the loop? Wouldn't this only ever be true or false for all cases? Could it be moved to the original `if`?

Reply:
removed and put it in the conditional